File: Maths_And_Stats/Statistics/Mean/meanOfGivenNumbers.cs
using System.IO;
using System;
using System.Linq;
class meanOfGivenNumbers
{
    static void Main()
    {
        var arr = new int[] { 10, 17, 25, 30, 40, 55, 60, 70 };
        double avg = Queryable.Average(arr.AsQueryable());
        Console.WriteLine("Average = " + avg);
    }
}

File: Maths_And_Stats/Statistics/Mean/meanOfGivenNumbers.js
var arr = [10, 17, 25, 30, 40, 55, 60, 70];
var total = 0;
for (var i = 0; i < arr.length; i++) {
    total += arr[i];
}
var avg = total / arr.length;
console.log("Average of numbers is " + avg)